### PR TITLE
[routing] add std::move in index_graph_starter_joints.hpp

### PR DESCRIPTION
**Pull request by gmoryes**
JTA

### CODE REVIEW

**Comment by gmoryes** [comment]
JTA

File: routing/index_graph_starter_joints.hpp [hunk 1]
```diff
@@ -549,7 +549,7 @@ std::vector<JointEdge> IndexGraphStarterJoints<Graph>::FindFirstJoints(Segment c
     result.emplace_back(fakeJoint, weight[beforeConvert]);
 
     std::vector<Segment> path = reconstructPath(beforeConvert, fromStart);
-    m_reconstructedFakeJoints[fakeJoint] = path;
```
Comment (vmihaylenko) [inline routing/index_graph_starter_joints.hpp]: emplace?

Comment (vmihaylenko) [reply routing/index_graph_starter_joints.hpp]:
```cpp
m_reconstructedFakeJoints(fakeJoint, reconstructPath(beforeConvert, fromStart));
```

Comment (gmoryes) [reply routing/index_graph_starter_joints.hpp]: dine

**Comment by gmoryes** [comment]
JTA

**Comment by gmoryes** [comment]
JTRIT

**Comment by gmoryes** [comment]
JTRQ

**Pull request merged**